Microsoft.Toolkit.Uwp.Connectivity/BluetoothLEHelper/ObservableGattCharacteristics.cs

```diff
@@ -152,6 +150,16 @@ public GattCharacteristic Characteristic
             {
                 if (_characteristic != value)
                 {
+                    if (value is not null)
+                    {
+                        value.ValueChanged += Characteristic_ValueChanged;
+                    }
+
+                    if (_characteristic is not null)
+                    {
+                        _characteristic.ValueChanged -= Characteristic_ValueChanged;
```

Comment:
nit: For clarity I'd unregister and then re-register, but it doesn't matter here since we're not doing this if the instance is the same.